pkg/cli/alpha/config-gen/testdata/componentconfig/expected.yaml

```diff
@@ -246,9 +101,9 @@ metadata:
     control-plane: controller-manager
 spec:
   ports:
-    - name: https
-      port: 8443
-      targetPort: https
+  - name: https
+    port: 8443
+    targetPort: https
```

Comment:
Why do we need to do these changes to bump 1.18?

Reply:
It was changed when I ran the tests. Without it, the teats don't work.